### PR TITLE
[#221] Add a label of minor version in backup.info

### PR DESCRIPTION
**Pull request by l32zhao**
I added a **MINOR_VERSION** label to record the minor version of the Postgres server in "backup.info" generated after making a full backup.

### CODE REVIEW

**Comment by jesperpedersen** [comment]
Only change what your patch is about... and use the same pull request

**Comment by l32zhao** [comment]
> Only change what your patch is about... and use the same pull request

Sorry about that, some codes were changed by Uncrustify. I have fixed it.

**Comment by Jubilee101** [comment]
Seems that some code got in without formatting @jesperpedersen

**Comment by Jubilee101** [comment]
@l32zhao What is your uncrustify version? We are using 0.77. Otherwise some code got in without formatting

**Comment by l32zhao** [comment]
> @l32zhao What is your uncrustify version? We are using 0.77. Otherwise some code got in without formatting

It's 0.76.0-2.fc38, I will try to upgrade it.

**Comment by l32zhao** [comment]
@Jubilee101 I upgraded uncrustify to 0.77.1 (Fedora39), but the result is still as same as my last commit, except for some modified codes not in my patch (like workers.h, workers.c, and bzip2_compression.c). Could you help me to have a look?

image

**Comment by Jubilee101** [comment]
Hi, that is ok. Could you revert the format changes that are not in your code? We'll submit a separate patch for formatting. Sorry for the trouble. Other than that, the code looks good to me~

**Comment by l32zhao** [comment]
> Hi, that is ok. Could you revert the format changes that are not in your patch? We'll submit a separate patch for formatting. Sorry for the trouble. Other than that, the code looks good to me~

Thanks, man. The current commit doesn't involve any format change not for adding the minor_version label. Please feel free to review it and pull it~

File: src/libpgmoneta/wf_backup.c [hunk 1]
```diff
@@ -135,6 +136,8 @@ basebackup_execute(int server, char* identifier, struct node* i_nodes, struct no
    }
    memset(version, 0, sizeof(version));
    snprintf(version, sizeof(version), "%d", config->servers[server].version);
+   memset(minor_version, 0, sizeof(minor_version));
+   snprintf(minor_version, sizeof(version), "%d", config->servers[server].minor_version);
```
Comment (jesperpedersen) [inline src/libpgmoneta/wf_backup.c:140]: minor_version

**Review by jesperpedersen — CHANGES_REQUESTED** [review]

**Comment by jesperpedersen** [comment]
Merged.

Welcome to the community !

Thanks for your contribution !

**Pull request merged**